resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -331,6 +332,18 @@ public DnsNameResolverBuilder ndots(int ndots) {
     }
 
     /**
+     * Set if domain / host names should be decoded to unicode when received.
+     * See <a href="https://tools.ietf.org/html/rfc3492">rfc3492</a>.
+     *
+     * @param decodeToUnicode if should get decoded
+     * @return {@code this}
+     */
+    public DnsNameResolverBuilder decodeToUnicode(boolean decodeToUnicode) {
```

Comment:
I'm thinking we should at least have an option to comply with the RFC (if not do this by default)